src/olympia/scanners/models.py

```diff
@@ -48,10 +54,12 @@ def add_yara_result(self, rule, tags=None, meta=None):
     def extract_rule_names(self):
         """This method parses the raw results and returns the (matched) rule
         names. Not all scanners have rules that necessarily match."""
-        if self.scanner is not YARA:
-            # We do not have support for other scanners yet.
-            return []
-        return sorted({result['rule'] for result in self.results})
+        if self.scanner is YARA:
```

Comment:
Had not spotted that before, but don't use `is` for integer value comparisons, use `==`. `is` is for identity comparisons, the fact that it works for integers up to 256 is an implementation detail.
```python
>>> a = 1
>>> a is 1
True
>>> a = 257
>>> a is 257
False
```

Reply:
oh I see, thanks